lib/Raven/Client.php

```diff
@@ -1506,4 +1512,23 @@ public function setReprSerializer(Raven_ReprSerializer $reprSerializer)
     {
         $this->reprSerializer = $reprSerializer;
     }
+
+    private function triggerAutoload()
+    {
+        // manually trigger autoloading, as it cannot be done during error handling in some edge cases due to PHP (see #60149)
+
+        if (! class_exists('Raven_Stacktrace')) {
+            spl_autoload_call('Raven_Stacktrace');
+        }
+
+        if (function_exists('mb_detect_encoding')) {
+            mb_detect_encoding('string');
+        }
+        if (function_exists('mb_convert_encoding')) {
+            mb_convert_encoding('string', 'UTF8');
+        }
+        if (function_exists('mb_convert_encoding')) {
```

Comment:
The fact that they are there twice is a mistake, I'll delete them ASAP.

As for the rest, functions are autoloaded too, so I need to trigger that in advance, to avoid triggering the autoload during the fatal error handling phase, where it would crash hard.

Reply:
Oh interesting! Where can I read about autoload for functions?  I thought you couldn't autoload functions..

Reply:
Sorry, I used the wrong term. Functions are not "autoloaded" like classes, but still they are searched by namespace and loaded with a similar logic. So calling a function for the first time only during error handling makes everything crash under PHP 5, and that's why I'm forced to do this workaround and call them at least once before proceeding.

Reply:
eesh! I had never seen a requirement to call functions first, but happily I don't use PHP 5 anymore so can't dig into this :)